homeassistant/components/zwave_js/climate.py

```diff
@@ -487,6 +523,13 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
         preset_mode_value = self._hvac_presets.get(preset_mode)
         if preset_mode_value is None:
             raise ValueError(f"Received an invalid preset mode: {preset_mode}")
+        # Dry and Fan preset modes are deprecated as of 2023.8
+        # Use Dry and Fan HVAC modes instead
+        if preset_mode_value in (ThermostatMode.DRY, ThermostatMode.FAN):
+            LOGGER.warning(
```

Comment:
We should probably add a repair issue too. It should be fixable and persistent and use warning level. See eg recent vacuum PRs where we added that.

Reply:
@MartinHjelmare - I'm not very familiar with repairs yet, but from what I've seen in other PRs, are you suggesting that instead of raising a simple warning, we could raise a repair issue which will be visible and actionable from the UI?  Should we keep the warning log as well?

Would something like this do the job?  Is it ok if we **only** raise it when somebody tries to set the affected preset modes?

```python
if preset_mode_value in (ThermostatMode.DRY, ThermostatMode.FAN):
    async_create_issue(
        hass,
        DOMAIN,
        "zwavejs_dry_fan_presets_deprecation",
        is_fixable=False,
        is_persistent=True,
        severity=IssueSeverity.WARNING,
        issue_domain=domain,
        translation_key="zwavejs_dry_fan_presets_deprecation",
        translation_placeholders={"domain": domain},
    )
```

Another question: you mentioned it should be fixable, but I'm not sure how to do it. Any hints in that front?

Once we clarify all this I'll raise a PR with the changes.

Reply:
Yes, a repair issue is enough instead of the warning. Just set `is_fixable` to `True` and make sure to add translation strings in strings.json.

Reply:
I can take this if you want Adrian